### PR TITLE
[MOB-3032] Release 1.1.0

### PR DESCRIPTION
**Pull request by EdwardW2**
## JIRA
[MOB-3032](https://karhoo.atlassian.net/browse/MOB-3032)

## DESCRIPTION
Back into develop

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [x] Other: Release

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [x] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [x] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

**Comment by EdwardW2** [comment]
Stable and canary versions uploaded to bintray

File: sdk/build.gradle [hunk 1]
```diff
@@ -72,7 +72,7 @@ android {
 publish {
     def groupProjectID = 'com.karhoo'
     def artifactProjectID = 'nsdk-canary'
-    def publishVersionID = '1.0.35.26102020-1'
+    def publishVersionID = '1.1.0.29102020-1'
```
Comment (mohadian) [inline sdk/build.gradle:75]: I would suggest to call it 1.1.0.29102020-RC-1

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Network SDK
 
+![](https://github.com/karhoo/karhoo-android-sdk/workflows/Master%20Branch%20CI/badge.svg) 
 ![](https://github.com/karhoo/karhoo-android-sdk/workflows/Develop%20Branch%20CI/badge.svg)
```
Comment (mohadian) [inline README.md:4]: Probably you need to remove the badge for the develop branch (workflow)

Comment (EdwardW2) [reply README.md:4]: Sure. Will remove that.

File: sdk/build.gradle (hunk 1 above)
Comment (EdwardW2) [reply sdk/build.gradle:75]: Alright, I will change and upload the version to the canary builds for testing.

File: sdk/gradlew [hunk 3]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (mohadian) [inline sdk/gradlew:1]: Can we rely on the Gradle files in the root folder?

File: sdk/gradlew.bat [hunk 4]
```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (mohadian) [inline sdk/gradlew.bat:1]: Can we rely on the Gradle files in the root folder?

File: sdk/gradlew (hunk 3 above)
Comment (mohadian) [reply sdk/gradlew:1]: same for gradle-wrapper.jar

Comment (EdwardW2) [reply sdk/gradlew:1]: This is auto-generated, my laptop and computer both have problems when trying to use gradle files in the project, so it always takes whatever the root gradle has

File: sdk/gradlew.bat (hunk 4 above)
Comment (EdwardW2) [reply sdk/gradlew.bat:1]: Should be able to, no problems with it so far.

**Review by mohadian — APPROVED** [review]

**Pull request merged**